### PR TITLE
OC-382 Update logback config

### PR DESCRIPTION
**Pull request by akoster**
Use logback version provided by Spring Boot

### CODE REVIEW

**Comment by CLAAdmin** [comment]
[CLA assistant check](http://cla.opensmartgridplatform.org:80/OSGP/open-smart-grid-platform?pullRequest=40) <br/>You should sign our Contributor License Agreement in order to get your pull request merged.<br/>

File: osgp/platform/osgp-logging/src/main/resources/logback.xml [hunk 1]
```diff
@@ -20,16 +20,16 @@
...
-    <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
-      <!-- daily rollover -->
-      <fileNamePattern>/var/log/osp/logs/osgp-logging.%d{yyyy-MM-dd}.log</fileNamePattern>
-
-      <!-- keep X days' worth of history -->
-      <maxHistory>10</maxHistory>
+    <rollingPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedRollingPolicy">
+      <fileNamePattern>${LOG_FULLNAME}.%d{yyyy-MM-dd mm}.%i.log</fileNamePattern>
+      <maxFileSize>100MB</maxFileSize>
+      <maxHistory>60</maxHistory>
```
Comment (kevinsmeets) [inline osgp/platform/osgp-logging/src/main/resources/logback.xml]: Change to 10

Comment (akoster) [reply osgp/platform/osgp-logging/src/main/resources/logback.xml]: Done.

**Comment by akoster** [comment]
> Update Logback version in the parent pom files:
> 
> ```
> open-smart-grid-platform/osgp/protocol-adapter-iec61850/parent-pa-iec61850/pom.xml:    <logback.version>1.1.2</logback.version>
> open-smart-grid-platform/osgp/shared/parent-shared/pom.xml:    <logback.version>1.1.2</logback.version>
> open-smart-grid-platform/osgp/platform/parent-platform/pom.xml:    <logback.version>1.1.2</logback.version>
> open-smart-grid-platform/osgp/protocol-adapter-iec60870/parent-pa-iec60870/pom.xml:    <logback.version>1.1.2</logback.version>
> open-smart-grid-platform/osgp/protocol-adapter-dlms/parent-pa-dlms/pom.xml:    <logback.version>1.1.2</logback.version>
> open-smart-grid-platform/osgp/protocol-adapter-oslp/parent-pa-oslp/pom.xml:    <logback.version>1.1.2</logback.version>
> open-smart-grid-platform/integration-tests/parent-integration-tests/pom.xml:    <logback.version>1.1.2</logback.version>
> ```

Done.

**Pull request merged**